model/logging/Logger.php

```diff
@@ -310,31 +336,69 @@ private function augumentMessageWithSessionId($message)
      */
     private function augumentMessageWithExecutionTime($message)
     {
-        $executionTime = number_format(microtime(true) - Controller::getExecutionTime(), 5);
+        $executionTime = $this->getExecutionTime();
 
         return "{$message} execution_time:'{$executionTime}'";
     }
 
     /**
-     * @param $level
-     * @param $message
+     * @return string
+     */
+    private function getExecutionTime()
+    {
+        return number_format(microtime(true) - Controller::getExecutionTime(), 5);
+    }
+
+    /**
+     * @param int|string $xframeLevel
+     * @param string $message
      *
      * @throws FrameEx
      */
-    private function logToDatabase($level, $message)
+    private function logToDatabase($xframeLevel, $message)
     {
         $log = new Record($this->tableName);
 
         $log->ip = $this->getIP();
         $log->key = $this->key;
-        $log->level = $level;
+        $log->level = $this->getDatabaseLevel($xframeLevel);
         $log->message = $message;
-        $log->date_time = date("Y-m-d H:i:s");
+        $log->date_time = $this->getDate();
         $log->session_id = session_id();
-        $log->execution_time = number_format(microtime(true) - Controller::getExecutionTime(), 5);
+        $log->execution_time = $this->getExecutionTime();
 
         $saveGraph = [];
         $log->save(false, $saveGraph, false);
```

Comment:
This is active record: https://github.com/assertis/xframe-legacy/pull/19/files#diff-18f35109564e166c073c399c0fa56a0dL326